matminer/featurizers/composition.py

```diff
@@ -254,6 +255,83 @@ def citations(self):
     def implementors(self):
         return ["Jiming Chen", "Logan Ward", "Anubhav Jain", "Alex Dunn"]
 
+class Meredig(BaseFeaturizer):
+    """
+    Class to calculate features as defined in Meredig et. al.
+    """
+
+    def __init__(self):
+        self.data_source = MeredigData()
+
+        self.features = [e.strip()+" fraction" for e in self.data_source.element_names]
```

Comment:
As "self.features" is only used internally by the featurizer, you should make it a protected variable. I.e., change the name from `self.features` to `self._features`.

Reply:
Changed